cuckoo_time_translator_algorithms/include/cuckoo_time_translator/Logging.h

```diff
@@ -0,0 +1,22 @@
+#ifndef CUCKOO_TIME_TRANSLATOR_LOGGING
```

Comment:
what should the header guard actually be?

Reply:
I kind of moved to UUID based guards to get rid of the nasty problem. I typically use Eclipse to generate them automatically. I believe it does an equivalent of this UNIX-like shell command
```bash
(echo -n H; uuidgen | tr  '[:lower:]-' '[:upper:]_')
```

A manual alternative is to copy one from an existing header and apply a small manual random disturbance ;).